addons/project/models/project_project.py

```diff
@@ -463,7 +460,8 @@ def write(self, vals):
         # directly compute is_favorite to dodge allow write access right
         if 'is_favorite' in vals:
             vals.pop('is_favorite')
-            self._fields['is_favorite'].determine_inverse(self)
+            for project in self:
+                project._set_favorite_user_ids()
```

Comment:
It's better to move the loop to `_set_favorite_user_ids`. It makes more sense that that method works for recordsets rather than only single records